src/main/java/org/sagebionetworks/bridge/sdk/LowercaseEnumDeserializer.java

```diff
@@ -0,0 +1,29 @@
+package org.sagebionetworks.bridge.sdk;
+
+import java.io.IOException;
+import java.lang.reflect.Method;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdScalarDeserializer;
+
+@SuppressWarnings("serial")
+class LowercaseEnumDeserializer extends StdScalarDeserializer<Enum<?>> {
+
+    protected LowercaseEnumDeserializer(Class<Enum<?>> clazz) {
+        super(clazz);
+    }
+
+    @Override
+    public Enum<?> deserialize(JsonParser jp, DeserializationContext ctxt) throws IOException, JsonProcessingException {
+        String text = jp.getText().toUpperCase();
+        try {
+            Method valueOfMethod = handledType().getDeclaredMethod("valueOf", String.class);
+            return (Enum<?>) valueOfMethod.invoke(null, text);
+        } catch (Exception e) {
+            throw new RuntimeException("Cannot deserialize enum " + handledType().getName() + " from " + text, e);
+        }
```

Comment:
RuntimeException is used here because higher up the call stack, we catch and rethrow a BridgeServerException which adds the endpoint that failed to the exception. We don't know that this far down in the method execution (and don't want to know). I can look for other "equivalent" exceptions though I'm not sure I know all the possible failure conditions here.